tools/wptrunner/wptrunner/executors/executoracacia.py

```diff
@@ -0,0 +1,91 @@
+import acacia_atspi
```

Comment:
We probably want to have eventually separate `executor` implementations per platform, and then a cross-platform implementation which can `import` the appropriate one and call its `get_accessibility_api_node` (or whatever API we go with) method.